docs/data/sdks/ios-swift/index.md

```diff
@@ -176,6 +181,45 @@ You can assign a new device ID using `deviceId`. When setting a custom device I
 amplitude.setDeviceId(NSUUID().uuidString)
 ```
 
+### Custom storage
+
+Every iOS app gets a slice of storage just for itself, meaning that you can read and write your app's files there without worrying about colliding with other apps. By default, Amplitude uses this file storage and creates an "amplitude" prefixed folder inside the app "Documents" directory. However, if you need to expose the Documents folder in the native iOS "Files" app and don't want expose "amplitude" prefixed folder, you can customize your own storage provider to persist events on initialization.
```

Comment:
Do you mean the prefix is ​​"amplitude"? Can we make it clearer?
https://github.com/amplitude/Amplitude-Swift/blob/main/Sources/Amplitude/Storages/PersistentStorage.swift#L24-L29

Or we can just say `If you don't want to store the data in the Amplitude-defined location, they can customize your own storage`.

Reply:
Yes, I want to say that the file name is prefixed with "amplitude".

> Or we can just say If you don't want to store the data in the Amplitude-defined location, they can customize your own storage.

I can put it as the first sentence but still would like to keep the rest to make it more clear where the location is and why do people want to use custom storage.